.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,37 @@
+---
+labels: bug
+name: "Bug Report"
+about: "Report an error condition or unexpected behavior"
+---
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+<!-- delete any unused steps -->
+1.
+2.
+3.
+4.
+
+## Context
```

Comment:
I think we are planning to use these templates for engineering team. In that case,  do we need the engineer to provide context as described here ?